src/components/notification/notificationPane.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { Component } from 'react';
+import { Grid, Box } from '@material-ui/core';
+import Divider from '@material-ui/core/Divider';
+import Popover from '@material-ui/core/Popover';
+import { connect } from 'react-redux';
+import {
+	GetNotifications,
+	markNotificationAsRead,
+	notificationPaneHeigthHandle,
+} from '../../actions/notificationPane';
+import { createBrowserHistory } from 'history';
+import moment from 'moment';
+
+export const history = createBrowserHistory({
+	forceRefresh: true,
+});
+
+export class NotificatonPane extends Component {
```

Comment:
@shemaeric1234 if this component is not managing any state, it's better to make it a functional component according to this project structure.

Reply:
Now this component is managing a state